packages/api/src/providers/__tests__/oauth1-deregister.test.ts

```diff
@@ -2,21 +2,26 @@
 import * as dotenv from "dotenv";
 dotenv.config({ path: ".env.test" });
 // Keep dotenv import and config before everything else
+import { nanoid } from "nanoid";
+import { v4 as uuidv4 } from "uuid";
 import * as updateProviderDataFile from "../../command/connected-user/save-connected-user";
 import * as getUserTokenFile from "../../command/cx-user/get-user-token";
 import * as saveUserTokenFile from "../../command/cx-user/save-user-token";
+import { makeConnectedUser } from "../../domain/__tests__/make-connected-user";
+import { makeUserToken } from "../../domain/__tests__/user-token";
 import { PROVIDER_GARMIN } from "../../shared/constants";
 import { OAuth1DefaultImpl } from "../oauth1";
-import {
-  userToken_2,
-  userToken_2_upd,
-  testUser_1,
-  testUser_1_upd,
-  testUser_2,
-  userToken_1_upd,
-  userToken_1,
-  testUser_2_upd,
-} from "./deregister-input";
+import { ProviderSource } from "@metriport/api-sdk";
+
+const userId = uuidv4();
+const cxId = uuidv4();
+const garminToken = nanoid();
+
+const userId_2 = uuidv4();
+const cxId_2 = uuidv4();
+const garminToken_2 = nanoid();
+
+const fitbitToken = nanoid();
```

Comment:
It's a good practice to create these for each test run, so we're not reusing IDs between executions - which could lead to a bug in the test/code to return data from a previous call and IDs would still match.

Could just make this a function that returns the IDs and we could deconstruct those in each test based on their needs - just an idea.